epsagon/utils.py

```diff
@@ -231,17 +234,21 @@ def init(
             os.getenv('EPSAGON_COLLECTOR_URL') or collector_url, token
         ),
         split_on_send=(
-                ((os.getenv('EPSAGON_SPLIT_ON_SEND') or '').upper() == 'TRUE')
-                | split_on_send
+            ((os.getenv('EPSAGON_SPLIT_ON_SEND') or '').upper() == 'TRUE')
+            | split_on_send
         ),
         propagate_lambda_id=(
-                ((os.getenv('EPSAGON_PROPAGATE_LAMBDA_ID') or '').upper() ==
-                 'TRUE')
-                | propagate_lambda_id
+            ((os.getenv('EPSAGON_PROPAGATE_LAMBDA_ID') or '').upper() ==
+             'TRUE')
+            | propagate_lambda_id
+        ),
```

Comment:
all this pycharm did automatically. if it passes the tests it it good enough or should be reverted?